software/nosql/src/main/java/brooklyn/entity/nosql/couchbase/CouchbaseClusterImpl.java

```diff
@@ -229,8 +229,13 @@ protected void doStart() {
                 }
                 
                 ((CouchbaseNode)getPrimaryNode()).rebalance();
-            } else if (getQuorumSize()>1) {
-                log.warn(this+" is not quorate; will likely fail later, but proceeding for now");
+            } else {
+                if (getQuorumSize()>1) {
+                    log.warn(this+" is not quorate; will likely fail later, but proceeding for now");
+                }
+                if (!serversToAdd.isEmpty()) {
+                    ((EntityInternal) Iterables.getOnlyElement(serversToAdd)).setAttribute(CouchbaseNode.IS_IN_CLUSTER, true);
```

Comment:
How do you know that `serversToAdd` will only have one element? If quorum size was 3, then presumably you could get into this code path with serversToAdd having size 2?